test/rmgpy/molecule/translatorTest.py

```diff
@@ -238,7 +239,7 @@ def test_ch2o2(self):
 3 O 1 {1,S}
 """
 
-        aug_inchi = "InChI=1/CH2O2/c2-1-3/h1H,(H,2,3)/u1,2"
```

Comment:
I think I'm not currently using InChIs for anything, but isn't an InChI changing kind of a big deal? Thought the whole point was they persisted!

Reply:
This change is actually caused by #2553 - could you copy this over there and tag @xiaoruiDong about this?

Reply:
I re-read the inchi discussion in that thread from October 2023, and don't want to follow up. Sounds like RDKit changed something but now match OpenBabel, so I'm gonna assume the change is a bug fix and let it go :)